pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -74,14 +74,14 @@ let
 in
 buildPythonPackage rec {
   pname = "tokenizers";
-  version = "0.20.2";
+  version = "0.20.3";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "huggingface";
     repo = "tokenizers";
     rev = "refs/tags/v${version}";
-    hash = "sha256-6uUlDLL1dMg5/6/z8AuB7ibqSJ7IUVexHDTkLx35wFY=";
+    hash = "sha256-NPH++kPPaSPR3jm6mfh+4aep6stj0I4bA24kFtaJSKU=";
   };
 
   cargoDeps = rustPlatform.importCargoLock { lockFile = ./Cargo.lock; };
```

Comment:
Why are we vendoring the lock file even though there are no git crates?

Reply:
I am not sure we can use `cargoHash` when using `buildPythonPackage` + `maturin` but I could be wrong on that.

Reply:
No but we can use `fetchCargoTarball`.

Reply:
No, it is indeed possible. I made the change.